baseplate/clients/redis_cluster.py

```diff
@@ -498,4 +498,37 @@ def execute_command(self, *args: Any, **kwargs: Any) -> Any:
     # pylint: disable=arguments-differ
     def execute(self, **kwargs: Any) -> Any:
         with self.server_span.make_child(self.trace_name):
-            return super().execute(**kwargs)
+
+            success = "true"
+            start_time = perf_counter()
+            labels = {
+                f"{PROM_LABELS_PREFIX}_command": "pipeline",
+                f"{PROM_LABELS_PREFIX}_client_name": self.connection_pool.connection_kwargs.get(
+                    "client_name", ""
+                ),
+                f"{PROM_LABELS_PREFIX}_database": self.connection_pool.connection_kwargs.get(
+                    "db", ""
+                ),
+                f"{PROM_LABELS_PREFIX}_type": "standalone",
+            }
+            num_reqs = len(self.command_stack)
+            commands = [cmd.args[0] for cmd in self.command_stack]
+
+            try:
+                with ACTIVE_REQUESTS.labels(**labels).track_inprogress():
+                    return super().execute(**kwargs)
+            except:  # noqa: E722
+                success = "false"
+                raise
+            finally:
+                ACTIVE_REQUESTS.labels(**labels).dec()
```

Comment:
the `with` block covers this case

Reply:
as i was rewriting this i was thinking i'd leave something behind, and that i did